### ISSUE TITLE
Put PluginManagerPlugins.xml on GitHub

### ISSUE THREAD

**Issue by sm4rk0** [bruderstein/nppPluginManager#6]
Hi Dave,
Would you consider this as it could make it easier for users to propose new plugins or updated versions? I couldn't find any way for doing this except using http://www.brotherstone.co.uk/npp/pm/admin/ which requires additional registration and is not transparent, or creating issues here. I feel PRs for PluginManagerPlugins.xml would better fit the GitHub usage scenario.

**Comment by bruderstein** [comment]
Hi, 
We did this originally, and had (at peak) about 6 contributors, and I ended up doing most of the updates manually. After introducing the admin site, we're up to over 50 contributors, and I don't do any manual updates, other than a few corrections (which take much less time than hand-editing XML).

I've just completed the next version of plugin manager, which enables us to automatically update the list from the admin system, which we can do on a fixed schedule. Publishing the changes to github can also be automated from there to gain transparency.

I'm closing this, as I don't wish to go back to doing most of the updates myself! (It's also a *lot* easier with the web front end!)
